src/main/java/com/coreos/jetcd/EtcdLease.java

```diff
@@ -68,6 +68,12 @@
     void closeKeepAliveService() throws IllegalStateException;
 
     /**
+     * It hints the state of the keep alive service.
+     * @return whether the keep alive service is running.
+     */
+    boolean isKeepAliveServiceRunning();
```

Comment:
Hi, should we have a function to imply the state of the keep alive service? @hongchaodeng